MANIFEST.in

```diff
@@ -3,3 +3,4 @@ include README.rst
 include CHANGELOG.rst
 include requirements/base.in
 include requirements/test.in
+graft edx_name_affirmation
```

Comment:
comparing to edx-proctoring's manifest, that also excludes some stuff under edx_proctoring (probably not relevant) and specifically excludes pyc (could be relevant):

global-exclude *.pyc

and includes everything out of requirements:

recursive-include requirements *.in *.txt

Reply:
I don't know if we really need all the requirements for production, but I can include them if we want to be on the safe side. (I actually don't think we need the test requirements either, going by other plugins like [platform-plugin-coaching](https://github.com/edx/platform-plugin-coaching/blob/master/MANIFEST.in))

As for excluding pyc, maybe? No idea on that one.

Reply:
certainly pycs will get made again so leaving them out is cleaner

doesn't seem to be a lot of standardization in these manifests though so ShipIt either way